platform/ios/Release-CN.md

```diff
@@ -0,0 +1,14 @@
+### Release `ios8-release-gazpacho`
+
+This branch is intended to exist as a branch off of of the main Maps SDK. There are no plans to bring this branch back into `master`. If this branch is updated to incorporate new release branches in the future, please update the branch name.
+
+Before running this script, please check that you are authorized to upload to S3 in China. 
+
+1. Update the `IOS_SDK_VERSION` value in the `platform/ios/scripts/package.sh` file. Versioning will follow: `vX.X.X-cn.X` in order to keep this distinct from the main Maps SDK.
```

Comment:
The regular release scripts [use the latest `ios-v*` git tag](https://github.com/mapbox/mapbox-gl-native/blob/3f314682fa2f2701c0d1c7e863013ce254a23afd/platform/ios/scripts/package.sh#L59) to determine the version string. In order for the proper version number to be set everywhere (e.g., in the framework’s plist, documentation, release notes, etc) and use a hard-coded version, you’ll need to overhaul everywhere that looks for a tag.